sunpy/map/mapbase.py

```diff
@@ -1572,6 +1578,31 @@ def draw_rectangle(self, bottom_left, width: u.deg, height: u.deg, axes=None, **
         else:
             axes_unit = self.spatial_units[0]
 
+        if width is not None or height is not None:
+            warnings.warn(SunpyDeprecationWarning("Use of `width` and `height` is depreciated and these will be removed in future"))
```

Comment:
```suggestion
            warnings.warn("Use of `width` and `height` is depreciated and these will be removed in future", SunpyDeprecationWarning)
```

Reply:
If this works, my understanding of how warnings/messages are raised needs to be fixed.